examples/waldo/README.md

```diff
@@ -48,6 +48,8 @@ cargo run --release --bin verify -- -i waldo.webp -r receipt.bin
 Running the verifier proves that the contents of [receipt.journal] were indeed constructed by the binary file associated with the expected [ImageID].
 
 [install Rust]: https://doc.rust-lang.org/cargo/getting-started/installation.html
+[receipt.journal]: https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Receipt.html#structfield.journal
```

Comment:
iiuc we're aiming to use tagged releases rather than `latest/` 
```suggestion
[receipt.journal]: https://docs.rs/risc0-zkvm/v0.18.0/risc0_zkvm/struct.Receipt.html#structfield.journal
```